db/migrate/20170506235850_create_conversations.rb

```diff
@@ -0,0 +1,10 @@
+class CreateConversations < ActiveRecord::Migration[5.0]
+  def change
+    create_table :conversations, id: :bigserial do |t|
```

Comment:
I know this came up on other PR, but - do we care at all about backfilling data? Or is the general approach here that we can just not care about it, and only things which exist after this gets deployed will have full conversation support?